scripts/aznfs_install.sh

```diff
@@ -183,73 +336,48 @@ esac
 
 ensure_pkg "wget"
 
+if [ "$RUN_MODE" == "auto-update" ]; then
+    parse_config_file
+
+    # Define the GitHub API URL to get the latest release
+    API_URL="https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/releases/latest"
```

Comment:
Also, I remember there is limitation on how many times this github api can be called per day/hour? If that is the case, then we have to think about that as well because it is running in loop and we don't know how many times it will get called.

Reply:
I have read about that, the rate-limit for github API for unauntheticated user is 60 times/hour. I have also verified that. Furthermore, it's okay for us to use this API as we'll be setting the frequency to check for updates once a day or once a week. We'll be running it in a loop, but the condition to run the update script will be true only once.

Reply:
If the limit is for per user, then it is fine. I thought total number of times this API can be called for a repo is restricted.